include/xnvme_dev.h

```diff
@@ -22,7 +22,9 @@ struct xnvme_dev {
 	struct xnvme_be be;       ///< Backend interface
 	struct xnvme_ident ident; ///< Device identifier
 
-	uint8_t _pad[4];
+	uint8_t _pad[2];
+	uint8_t is_identified; ///< Device is identified
+	uint8_t is_derived;    ///< Device geometry is derived
```

Comment:
Perhaps these should be booleans - for the sake of clarity.

Reply:
Agree, did this and...